controller_manager/src/controller_manager.cpp

```diff
@@ -1716,6 +1766,15 @@ void ControllerManager::RTControllerListWrapper::wait_until_rt_not_using(
   }
 }
 
+std::pair<std::string, std::string> ControllerManager::split_command_interface(
```

Comment:
Can we move this in utils namespace of `hardware_interface/handle.hpp` since there is also defined a rule that interfaces have "/" in their name. What do you think?

Reply:
I think we discussed this at some point. It'd be best to have these values pre-computed and served through 2 getters

Reply:
@bmagyar  Don't the service callbacks run on the non-real-time thread though? So I don't think will be a problem here.

Reply:
yeah that's ok but I meant this may be useful elsewhere in RT code too but let's go ahead for now